fxmanifest.lua

```diff
@@ -1,11 +1,11 @@
 fx_version 'cerulean'
 game 'gta5'
 
-description 'https://github.com/QBCore-Remastered'
-version '1.0.1'
+description 'https://github.com/Qbox-project'
+version '1.0.2'
```

Comment:
I don't think this resource is versioned yet. We should set the version to 1.0.0 instead.
```suggestion
version '1.0.0'
```